docs/detections/rules-ui-create.asciidoc

```diff
@@ -275,38 +283,40 @@ field values.
 === Preview your rule (optional)
 
-You can preview any rule to learn how noisy it will be before saving it. If necessary, you can then adjust the rule's query.
+You can preview any custom or prebuilt rule to find out how noisy it will be. For a custom rule, you can then adjust the rule's query or other settings.
 
 NOTE: To preview rules, you need the `read` privilege to the `.preview.alerts-security.alerts-<space-id>` index and `All` privileges for the Security feature. Refer to <<detections-permissions-section>> for more information.
 
-To preview a rule:
+Click the *Rule preview* button while creating or editing a rule. The preview opens in a side panel, showing a histogram and table with the alerts you can expect, based on the defined rule settings and past events in your indices. 
 
-. Write the rule query.
-. Choose how you want to preview the query results:
+[role="screenshot"]
+image::images/preview-rule.png[Rule preview]
 
-** *Quick query preview*: Select from pre-defined time frames -- *Last hour*, *Last day*, or *Last month* -- when previewing rule results. Note that threshold and event correlation rules have limited time frame options. The rule interval and look-back time are also pre-defined for the preview and differ by rule type. These settings cannot be modified.
-** *Advanced query preview*: Choose a custom time frame for the rule preview, schedule how often the rule should run, and specify a look-back time.
-+
-TIP: Avoid setting long time frames with short rule intervals. This might cause the rule preview to timeout.
+The preview also includes the effects of rule exceptions and override fields. In the histogram, alerts are stacked by `event.category` (or `host.name` for machine learning rules), and events with multiple values are counted more than once.
 
-. Click *Preview results*. The rule preview shows a histogram and alerts table with the alerts you can expect, based on the defined rule parameters and historical events in your indices. You can view the details of a particular alert by clicking the *View details* button in the alerts table.
-+
-NOTE: The preview excludes the effects of rule exceptions and timestamp overrides. In the preview histogram, alerts are stacked by `event.category` (or `host.name` for machine learning rules), and events with multiple values are counted more than once.
+To interact with the rule preview:
+
+* Use the date and time picker to define the preview's time range.
 +
-[role="screenshot"]
-image::images/preview-rule.png[]
+TIP: Avoid setting long time ranges with short rule intervals, or the rule preview might time out.
+
+* Click *Refresh* to update the preview. 
+** When you edit the rule's settings or the preview's time range, the button changes from blue (image:images/rule-preview-refresh-circle.png[Blue circular refresh icon,16,17]) to green (image:images/rule-preview-refresh-arrow.png[Green right-pointing arrow refresh icon,17,17]) to indicate that the edits might affect the preview.
```

Comment:
Using "might affect" here sounds to me like we're not sure if the edits get included in the preview or not. Would something along the lines of "The button changes from blue to green to indicate that the rule has been edited since the last preview" work here?

Reply:
```suggestion
** When you edit the rule's settings or the preview's time range, the button changes from blue (image:images/rule-preview-refresh-circle.png[Blue circular refresh icon,16,17]) to green (image:images/rule-preview-refresh-arrow.png[Green right-pointing arrow refresh icon,17,17]) to indicate that the rule has been edited since the last preview.
```